app/models/team.rb

```diff
@@ -182,6 +184,10 @@ def without_preferences?(att)
     att[:conference_id].blank?
   end
 
+  def without_conferences?(att)
```

Comment:
👍 
Maybe use `attributes` instead of `att` - would make it more obvious on first glance (same for the `without_preferences` method).

In this case, given the size of the class, I would actually even suggest to use the direct `proc` syntax, otherwise one has to scroll to the bottom of the large file to understand what's going on... What do you think?
```ruby
accepts_nested_attributes_for :conference_attendances, allow_destroy: true, reject_if: proc { |attributes| attributes[:conference_id].blank? }
```

But totally up to you, does not change anything in behavior.